modular_zubbers/code/modules/antagonists/bloodsucker/bloodsuckers/life.dm

```diff
@@ -104,6 +106,8 @@
 	var/fireheal = 0 // BURN: Heal in Coffin while Fakedeath, or when damage above maxhealth (you can never fully heal fire)
 	// Checks if you're in a coffin here, additionally checks for Torpor right below it.
 	var/amInCoffin = istype(user.loc, /obj/structure/closet/crate/coffin)
+	if (blood_over_cap > 0)
+		mult = 1 + round(blood_over_cap / 1000, 0.1) // effectively 1 (normal) + 0.1 for every 100 blood you are over cap
```

Comment:
keep in mind mult is a healing multiplier, also you are overriding it here which fucks over things that increase it.

Reply:
Changed to scale off of costmult instead